src/commands/mod.rs

```diff
@@ -581,6 +585,10 @@ struct MoveArgs {
 /// If the source became empty and both the source and destination had a
 /// non-empty description, you will be asked for the combined description. If
 /// either was empty, then the other one will be used.
+///
+/// Note that if squashing working copy commit results in abandoning of empty commit,
+/// a new empty commit will still be created as a working copy commit.
+/// If this is undesirable, use jj unsquash.
```

Comment:
Maybe like this instead:

> To edit the resulting squashed commit, use `jj edit -r @-`.

Reply:
For me personally your wording does not address my original confusion. If the help was worded like this, I wouldn't be certain that this will help me. I will try another way